basic-caching/docs/cloudshell-tutorial.md

```diff
@@ -0,0 +1,90 @@
+# Basic Caching
+
+---
+This sample shows how to implement a caching using Apigee's [ResponseCache](https://cloud.google.com/apigee/docs/api-platform/reference/policies/response-cache-policy), [PopulateCache](https://cloud.google.com/apigee/docs/api-platform/reference/policies/populate-cache-policy), and [LookupCache](https://cloud.google.com/apigee/docs/api-platform/reference/policies/lookup-cache-policy) policies.
+
+Let's get started!
+
+---
+
+## Setup environment
+
+Ensure you have an active GCP account selected in the Cloud shell
+
+```sh
+gcloud auth login
+```
+
+Navigate to the `basic-caching` directory in the Cloud shell.
+
+```sh
+cd basic-caching
+```
+
+Set the following variables:
+
+* `PROJECT` the project where your Apigee organization is located
+* `APIGEE_HOST` the externally reachable hostname of the Apigee environment group that contains APIGEE_ENV
+* `APIGEE_ENV` the Apigee environment where the demo resources should be created
+
+For example, you can use a command such as the following:
+
+Set the project ID:
+```sh
+export PROJECT=<replace with project ID>
+```
+
+Set the DNS host name where the proxy can be reached:
+```sh
+export APIGEE_HOST=<replace with host name>
+```
+
+Set the environment name where the proxy will be deployed:
+```sh
+export APIGEE_ENV=<replace with target environment name>
+```
+---
+
+## Deploy Apigee components
+
+Next, let's create and deploy the Apigee resources necessary to test the caching policies.
+
+```sh
+./deploy-basic-caching.sh
+```
+
+This will not only deploy the proxy but also run the tests..
```

Comment:
remove extra trailing `.`

Reply:
Corrected in [e6a6bf4](https://github.com/GoogleCloudPlatform/apigee-samples/pull/67/commits/e623d016e9a063cf5f38696a7a7008b18e6a6bf4)